Developer_readme.md

```diff
@@ -0,0 +1,237 @@
+
+          isadvisable = function() -- logic that tells svo when to put this defence
+            return (not defc.dragonform and not defc.devour and ((sys.deffing and defdefup[defs.mode].devour) or (conf.keepup and defkeepup[defs.mode].devour)) and not codepaste.balanceful_defs_codepaste() and not affs.paralysis and not affs.prone and (defc.mouths and defc.tentacles) and bals.anathema) or false
+          end,
+          -- ^ means svo will only put this defence if the user is not dragon, don't already have devour up, is not in the process of putting up or keeping up a defence at the time, has passed balanceful defs checks, is not paralysed and not prone, has the mouths and tentacles defences already up and has anathema balance available. 
+
+          oncompleted = function() defences.got('devour') end, -- defence is sucessfully up? tell svo you got it.
+
+          action = "unnamable devour", -- defines the action that will put this defence up
+          onstart = function() -- logic to put the defence up, add what needs to be sent and any other applicable logic to it here.
+            send('unnamable devour', conf.commandecho) 
+            end
+        }
+      }
+    ```
+    
+3. add its entry in the defences dataset (defs_data) in **svo (alias and defence functions) > Defences**  with all the on/off lines as well as the appropriate configurations.
+<details>
+    <summary>For example, you want to add entries for defs of a class specific skill, let's see how Necromancy defences are set, with comments!</summary>
+    
+```lua
+if svo.haveskillset('necromancy') then --this is important, as you don't want the defence list to be cluttered with a lot of defences that the user's current class cannot put up! Always check if the skillset is available before populating it!
+  defs_data:set('deathsight', { type = 'necromancy', -- the type determines from what skillset the defence will be put into, usually you want the type to be the same name as the skillset you are adding it for
+    staysindragon = true, -- means that the defence will stay on even after dragonforming
+    availableindragon = true, -- means it can be putup while in dragonform. For this specific case, 'deathsight' is also a general defence that can be put up in dragonform so this flag this defence so svo to not unnecessarily drop it after the defence table is repopulated upon dragonforming
+    def = "Your mind has been attuned to the realm of Death.", -- the line that appears when you check DEF ingame
+    on = {"Your mind is already attuned to the realm of Death.", "You shut your eyes and concentrate on the Soulrealms. A moment later, you feel inextricably linked with realm of Death."}, -- these are the lines that are fired when you put the defence up. 
+    onr = "^A miasma of darkness passes over your eyes and you feel a link to the realm of Death,? form in your mind\\.$", --same as above, except that it uses regex. Use onr anytime you need regex instead of a normal trigger.
```

Comment:
Should this be just one `\`?

Reply:
umm i dunno, I copied it straight from the code itself! Maybe it was a typo?